cmd/networkPolicies_test.go

```diff
@@ -0,0 +1,141 @@
+package cmd
+
+import (
+	"testing"
+
+	apiv1 "k8s.io/api/core/v1"
+	networking "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// TODO test default deny
```

Comment:
I have seen a couple of TODOs in your code, are you intending to address them before we get this merged?